python/cloud-run-django-hello-world/.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+// launch.json contains information needed for the debugger to attach to  processes
+// See https://code.visualstudio.com/Docs/editor/debugging#_launch-configurations
+// for documentation about how to create and manage configurations
+{
+  "version": "0.2.0",
```

Comment:
Again, flagging this file as we'll need changes here but I don't have the exact changes yet.